packages/ifpack2/src/Ifpack2_Details_CrsArrays.hpp

```diff
@@ -125,7 +125,6 @@ struct CrsArrayReader
     KOKKOS_INLINE_FUNCTION void operator()(const int row) const
     {
       LocalOrdinal offset = rowptrs(row);
-      std::cout << "Rowptr[" << row << "] = " << offset << '\n';
       size_t entries = rowptrs(row + 1) - offset;
       Teuchos::Array<Scalar> valsArray(entries);
```

Comment:
* Don't use `Teuchos::Array` in a device function.
* If you won't ever run this on CUDA, then don't use the `KOKKOS_INLINE_FUNCTION` macro -- that will avoid build errors and warnings.